spec/models/storage_location_spec.rb

```diff
@@ -22,6 +22,19 @@
     end
   end
 
+  context "Callbacks >" do
+    describe "before_destroy" do
+      let!(:item) { create(:item) }
+      subject { create(:storage_location, :with_items, item_quantity: 10, item: item, organization: @organization) }
```

Comment:
The subject shouldn't refer itself to the `.destroy` function? I mean, the tested one.